image_writers_guide/guidelines.adoc

```diff
@@ -9,4 +9,135 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+When creating Docker images to run on OpenShift there are a number of best practices to consider in order to ensure consumers of those images have a good experience.  Since images are intended to be immutable and used as is, it is the image author's responsibility to ensure the image contains any artifacts that would typically be needed by a user of the image and that the image can be configured by the user.  Here are some general comments around things to consider.
+
+== General Docker Best Practices
+The following are some guidelines that apply to Docker image creation in general, independent of whether the images are used on OpenShift v3.
```

Comment:
In the general section you mention a lot of OpenShift specific solutions - for example how volumes work on OS. I guess this could be split into two sections and move the OS specific part down and link to it.